packages/app/run_ios.sh

```
@@ -0,0 +1,60 @@
+  ls $HOME/Projects/ratel/target/x86_64-apple-ios/debug/
+  if [ -f "$SRC" ]; then
+    mkdir -p "$(dirname "$DEST")"
+    cp "$SRC" "$DEST"
+    echo "Copied: $SRC → $DEST"
+  else
+    echo "Missing: $SRC"
+  fi
+done
+
+echo "📦 Installing CocoaPods (if needed)..."
+cd "$HOME/Projects/ratel/packages/app/gen/apple"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)